docs/example.md

```diff
@@ -112,7 +130,7 @@ This has been tested on ADB 1.8.0 with Fedora 23 on host machine
 
  1. Label /var/lib/kubelet/pods as `svirt_sandbox_file_t`
     This step is because of bug [#117](https://github.com/projectatomic/adb-atomic-developer-bundle/issues/117) in ADB 
-    - `chcon -Rt svirt_sandbox_file_t /var/lib/kubelet/pods`
+    - `sudo chcon -Rt svirt_sandbox_file_t /var/lib/kubelet/pods`
 
  1. Run mlbparks Nulecule  app on Kubernetes.
     - `sudo -E atomic run 10.1.2.1:5000/mlbparks-atomicapp`
```

Comment:
I added note about that IP to previous step:
`If you skipped first step, because you are using you own Docker registry replace '10.1.2.1' with address of your registry.`